src/libraries/Microsoft.PowerFx.Core/Syntax/NamedFormulas.cs

```diff
@@ -79,17 +80,17 @@ public IEnumerable<TexlError> GetParseErrors()
         }
 
         /// <summary>
-        /// Returns a Formula object for each named formula.
+        /// Returns a Tuple of a DName and Formula object for each named formula.
         /// </summary>
         /// <returns></returns>
-        public IEnumerable<Formula> GetFormulas()
+        public IEnumerable<Tuple<DName, Formula>> GetNamedFormulas()
```

Comment:
C# has some nice things now like named tuples. This type could be 
`IEnumerable<(DName name, Formula formula)>`, and then instead of Tuple.Create, just do `formulas.Add((kvp.key, GetFormula(kvp.Value)))`

Reply:
see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples

Reply:
Updated